elastic/beats
# Removed_beta_Warning flag from IIS Application pool

- Bug

## What does this PR do?

Closes #35061 

## Why is it important?

The IIS application pool datastream is already moved to GA but the logger warning is still pointing to BETA.

## How to test this PR locally

Enable IIS and application pool stream and check whether the logger warns with BETA flag.
